_posts/2019-01-25-best-practices-for-building-web-appbuilder-widgets.md

```diff
@@ -9,18 +9,27 @@ categories:
-* a best practice folder structure
-* grunt tasks for JS tooling including babel, livereload, and sass
-* a grunt task to automate the copying of your development code to WAB developer edition
+* a best practice folder structure;
+* grunt tasks for JS tooling, including babel, livereload, and sass; and
+* a grunt task to automate the copying of your development code to WAB developer edition.
 
 Esri also includes a generator for scaffolding out the widgets themselves. This project provided a great starting point for my work.
 
-Another great tip that I found was to write your WAB widgets as plain old dojo widgets (inherit from `dijit._WidgetBase`) and then wrap them for WAB. This made unit tests much easier. The other benefit to this approach is that it makes it much easier to reuse the widgets in non-WAB applications. Here's an example repo: [gbochenek/wab-test-example](https://github.com/gbochenek/wab-test-example).
+### 2. Write your WAB widgets as dojo widgets.
+Another great tip that I found was to write your WAB widgets as plain old dojo widgets (inherit from `dijit._WidgetBase`) and then wrap them for WAB. This made unit tests much easier. The other benefit to this approach is that it makes it much easier to reuse the widgets in non-WAB applications. Here is an example repo: [gbochenek/wab-test-example](https://github.com/gbochenek/wab-test-example).
 
-Using babel worked great for this project until I needed polyfills for older browsers. I'm looking at you, IE. Loading third party libraries via AMD from within WAB widgets [is a pain](https://developers.arcgis.com/web-appbuilder/sample-code/add-a-third-party-library.htm). Fortunately, I found [Polyfill.io](https://polyfill.io/v3/). This is a service that dynamically returns a set of polyfills as defined by query parameters. The cool part about this service is that it only loads the code required for the browser that's making the request. Using this library from within WAB requires [adding it to the `loadResource` method in `init.js`](https://github.com/agrc/wfrc-wab-widgets#add-a-polyfill-for-older-browsers).
+### 3.Use Polyfill.io.
+Using babel worked great for this project until I needed polyfills for older browsers. (I'm looking at you, Internet Explorer.) Loading third-party libraries via AMD from within WAB widgets [is a pain](https://developers.arcgis.com/web-appbuilder/sample-code/add-a-third-party-library.htm). Fortunately, I found [Polyfill.io](https://polyfill.io/v3/). This is a service that dynamically returns a set of polyfills as defined by query parameters. The cool part about this service is that it only loads the code required for the browser that is making the request. Using this library from within WAB requires [adding it to the `loadResource` method in `init.js`](https://github.com/agrc/wfrc-wab-widgets#add-a-polyfill-for-older-browsers).
 
 I hope that these tips save you some headaches in the future. {% capture contact %}{% include contact.html
```

Comment:
Conclusions/wrapping things up are always so difficult. Bleh. But I feel like maybe there's room for just a little more easing into the contact info here at the end? Can we add something to kind of sum things up and transition into the closing more smoothly?